core/array/shovel_append_spec.rb

```diff
@@ -0,0 +1,35 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require File.expand_path('../fixtures/classes', __FILE__)
+
+describe "Array#<<" do
```

Comment:
I think we should just keep this in the `core/array/append_spec.rb` file, it's OK to have specs for multiple methods in the same file if the [naming convention](https://github.com/ruby/mspec/blob/master/lib/mspec/utils/name_map.rb) says they should be there. There are already other cases like this I believe, notably `Foo.bar/Foo#bar`.
Could you move these specs just above Array#append's specs in  `core/array/append_spec.rb`?